recipe/build-r-base.sh

```diff
@@ -540,9 +540,10 @@ Darwin() {
     make install ${EXTRA_MAKE_ARGS}
 
     pushd ${PREFIX}/lib/R/etc
-      sed -i -r "s|-isysroot ${CONDA_BUILD_SYSROOT}||g" Makeconf
-      sed -i -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf
-      sed -i -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf-r
+      sed -i'.bak' -r "s|-isysroot ${CONDA_BUILD_SYSROOT}||g" Makeconf
+      sed -i'.bak' -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf
+      sed -i'.bak' -r "s|-lemutls_w|-lgcc_s.1.1|g" Makeconf
```

Comment:
Thanks go out to @isuruf for, among other things, explaining to me that linking to static libs could cause issues for R packages that use both Fortran and C++ since C++ exception handling across shared lib bounds should use the same shared libs!